Aztec/Classes/TextKit/MediaAttachment.swift

```diff
@@ -32,7 +32,7 @@ open class MediaAttachment: NSTextAttachment {
 
     /// Identifier used to match this attachment with a custom UIView subclass
     ///
-    private(set) open var identifier: String
+    open var identifier = String()
```

Comment:
This is only used by this class... and there's no public method offering any interaction with it whatsoever...

Why the change?

Reply:
Because i've been dealing with the idea of switching this to a `let` constant, ended up reverting... and forgot it was `private(set)`. Nice catch!